TTPChallenge/DataAnalystVC.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  DataAnalystVC.swift
+//  TTPChallenge
+//
+//  Created by Mirim An on 2/15/17.
+//  Copyright © 2017 TeamMDC. All rights reserved.
+//
+
+import UIKit
+
+class DataAnalystVC: UIViewController {
+
+    @IBOutlet var informationButtons:[UIButton]!
+    
+    @IBOutlet weak var retakeQuizButton: UIButton!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        for button in informationButtons {
+            button.layer.cornerRadius = 12
```

Comment:
Followed the step and it's not working - no changes onto the buttons. Why is it better to do this way?